Dockerfiles/manifests/openshift/scc.yaml

```diff
@@ -42,10 +42,18 @@ defaultAddCapabilities: []
 fsGroup:
   type: MustRunAs
 readOnlyRootFilesystem: false
-runAsUser:
+runAsUser: 
   type: MustRunAsRange
 supplementalGroups:
   type: RunAsAny
+# If your environment restricts user access to the Docker socket, 
+# you can add a group under `supplementalGroups` that has access
+# by uncommenting the following lines (and removing the previous line `type: RunAsAny`):
```

Comment:
If I remember well how RHEL and OpenShift are working, by default, there is no `docker` group defined on the system and the docker socket belongs to `root:root`.
The end-user needs to create a `docker` group (with `groupadd`) and restart the docker daemon so that it recreates a socket that belongs to `root:docker`.
Once this has been done, the SCC can be enhanced to add the `SupplementalGroups`.

I don’t think it is desirable to have the `docker` group creation done by the Datadog deployment. But it could be smart to put in the comment the fact that the end-user has this setup to do on the host before hand.

Reply:
Ah, good point. I'll generalize this section a bit more as this is dependent on the environment and should be managed by the user according to their setup.